core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -101,6 +103,21 @@ __global__ __launch_bounds__(
     maxTperB,
     minBperSM) static void hip_parallel_launch_local_memory(const DriverType
                                                                 *driver) {
+  // FIXME_HIP driver() pass by copy
+  driver->operator()();
+}
+
+template <typename DriverType>
+__global__ static void hip_parallel_launch_global_memory(
+    const DriverType *driver) {
+  driver->operator()();
+}
+
+template <typename DriverType, unsigned int maxTperB, unsigned int minBperSM>
+__global__ __launch_bounds__(
```

Comment:
same thing. Please comment that these are currently identical to the analogous Cuda versions and comment on the Cuda versions that those are identical to the HIP versions